sherpa/__init__.py

```diff
@@ -80,12 +81,12 @@ def get_config():
 
     # If NOSHERPARC is set, read in system config file
     # ignore any user config file
-    if (os.environ.has_key('NOSHERPARC') == True):
+    if (('NOSHERPARC' in os.environ) == True):
```

Comment:
This (and the following case) can be further cleaned up to

    if 'NOSHERPARC' in os.environ: